test/cases/regression-no-explicit-rootfs-definition.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# https://bugzilla.redhat.com/show_bug.cgi?id=2049500
+
+# Get OS data.
+source /usr/libexec/osbuild-composer-test/set-env-variables.sh
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh
+
+case "${ID}-${VERSION_ID}" in
+    "rhel-8.6" | "rhel-9.0" | "centos-9")
```

Comment:
Why not run it on `centos-8` too? And we want to only run them on those specific RHEL versions?

Reply:
I think this was a copy & paste error. I've removed the entire case block.